tests/plugins/elementspath/_helpers/tools.js

```diff
@@ -19,9 +19,38 @@ var elementspathTestsTools = ( function() {
 		expected = expected.concat( tags.split( ',' ) );
 
 		assert.areEqual( expected, list.join( ',' ), 'Invalid elements path.' );
-	};
+	}
+
+	/*
+	 * Fires element event handler attribute e.g.
+	 * ```html
+	 * <button onkeydown="return customFn( event )">x</button>
+	 * ```
+	 *
+	 * @param {CKEDITOR.dom.element/HTMLElement} element Element with attached event handler attribute.
+	 * @param {String} eventName Event handler attribute name.
+	 * @param {Object} evt Event payload.
+	 */
+	function fireElementEventHandler( element, eventName, evt ) {
```

Comment:
Helper copied and moved to `bender.tools.fireElementEventHandler`. More details: https://github.com/ckeditor/ckeditor-dev/pull/2828